# generated by patch-package 6.4.10
#
# command:
#   npx patch-package next
#
# declared package:
#   next: 13.4.19
#
diff --git a/node_modules/next/image-types/global.d.ts b/node_modules/next/image-types/global.d.ts
index 661c316..aac36d7 100644
--- a/node_modules/next/image-types/global.d.ts
+++ b/node_modules/next/image-types/global.d.ts
@@ -8,12 +8,9 @@ declare module '*.png' {
 }
 
 declare module '*.svg' {
-  /**
-   * Use `any` to avoid conflicts with
-   * `@svgr/webpack` plugin or
-   * `babel-plugin-inline-react-svg` plugin.
-   */
-  const content: any
+  import { FC, SVGProps } from 'react'
+
+  const content: FC<SVGProps<SVGSVGElement>>
 
   export default content
 }
